_includes/coffee/simulation.coffee

```diff
@@ -701,7 +733,12 @@ build = (data, sim_name, codes_data, chart_data, axes_names, repo_slug) ->
   set_repo(data)
 
   result_data = groupBy(((x) -> x.type), data.data)
-  vega_data = vegarize(chart_data, axes_names)(result_data.line)
+
```

Comment:
What if the expected `benchmark_data` wasn't uploaded? *E.g.*, featured plot is free energy but the prankster only uploads pictures?

Reply:
Nothing breaks, it just doesn't show anything in the plot showing the free_energy versus time graph but without any data. Likewise for the result comparison page. It's difficult to take care of this during the linting.